src/iosrv/srv.c

```diff
@@ -266,6 +268,24 @@ check_sleep_list()
 	}
 }
 
+/**
+ * sleep micro seconds, then being rescheduled.
+ * \param[in]	us	micro seconds to sleep for
+ */
+int
+dss_sleep(uint64_t sleep_usec)
+{
+	struct dss_sleep_ult *dsu;
+
+	dsu = dss_sleep_ult_create();
```

Comment:
I'm afraid that's too much memory allocation, I prefer keeping the original dss_sleep().

Reply:
The original dss_sleep using yield will be kept scheduling. though I agree memory allocation is not very good either, but since it is a bit memory, and most of dss_sleep is not in critical hot path. so it is probably ok, since we do not need 2 sleep anyway. we probably can optimize the allocation later.